spec/unit/remote_system_spec.rb

```diff
@@ -18,7 +18,15 @@
 require_relative "spec_helper"
 
 describe RemoteSystem do
-  let(:remote_system) { RemoteSystem.new("remotehost") }
+  let(:rsys) { RemoteSystem.new("remotehost") }
```

Comment:
all these initializations of `RemoteSystem` would be better handled by using the subject and then putting the different arguments in variables that describe what they are ment to be. This way you could always use the subject and depending on the describe/context block define those argument variables